fendermint/app/options/src/rpc.rs

```diff
@@ -92,6 +92,10 @@ pub enum RpcQueryCommands {
     },
     /// Get the slowly changing state parameters.
     StateParams,
+    /// Query the current upgrade schedule
+    UpgradeSchedule,
```

Comment:
I am not exactly sure what an upgrade schedule is; maybe the docstring could explain a bit more. 

For example I understand it can be the contents of the JSON file, but I was wondering if it could also include the versions that the node is capable of executing.